app/webpacker/src/javascripts/commodities.js

```diff
@@ -535,7 +434,7 @@
         @description adds behaviours
         @param {Element} content Element in which to add behaviours
       */
-    onLoad: function(context) {
+    onLoad: function (context) {
       if (context === undefined) {
         context = document.body;
       }
```

Comment:
Looks good but please can you remove `this.countryPicker.initialize();` from line 445 as it throws a js error, thanks. :) 
![image](https://github.com/user-attachments/assets/35f8a988-2ebc-494f-9838-7404eb7c1400)